source/webapp-cards-2.00-ru.yaml

```
@@ -433,16 +433,14 @@ suits:
     id: "JOA"
     value: "A"
     url: "https://cornucopia.owasp.org/cards/JOA"
-    #card: "Джокер"
-    card: "Joker"
+    card: "Джокер"
```

```suggestion
    #card: "Джокер"
    card: "Joker"
```
I agree that this is the right translation, but it won't look good because of the length of these characters. We will have to reduce the font size before we can do this. For now, we should just leave it commented out.

---

At the moment we don't translate the "J", "Q", "K" or "A" either. If we decide to change "Joker" to be language-specific, will these other letters printed on cards are also appear translated?

For example, this page https://en.wikipedia.org/wiki/Russian_playing_cards suggests we need В, Д, К and T for Russian, but other languages may need space for two letters: https://en.wikipedia.org/wiki/French-suited_playing_cards

---

Good point, no currently we use the values from the mapping file. I had a brief look at this, but didn't go further into it.

---

> I agree that this is the right translation, but it won't look good because of the length of these characters. We will have to reduce the font size before we can do this. For now, we should just leave it commented out.

Good catch. I forgot to revert the translation for Joker. I saw your patch for this issue.